contracts/cloud.yml

```diff
@@ -1198,41 +1198,45 @@ paths:
             $ref: '#/components/schemas/WritePrecision'
       responses:
         '204':
-          description: Write data is correctly formatted and accepted for writing to the bucket.
+          description: 'The sent data was correctly formatted and accepted for writing to the bucket. `204` doesn''t indicate a successful write operation since writes are asynchronous. See [how to check for write errors](###some-link###).'
```

Comment:
Waiting on the docs.influxdata.com content for the feature described in https://github.com/influxdata/docs-v2/issues/3003

Reply:
I agree with this comment - https://github.com/influxdata/docs-v2/issues/3003#issuecomment-906787335, `202` feels more appropriate to add as a response, until then, this description update suffices.